peach & peach with .Q.MAP (#177)
KxSystems/docs: docs/ref/each.md

```

`each` and `peach` perform the same computation and return the same result. 

`peach` will divide the work between available secondary tasks. 

> !!! note Since 4.1t 2024.01.04 peach workload distribution methodology changed to dynamically redistributes workload and allows nested invocation. 

Reviewer: ```suggestion
!!! note Since 4.1t 2024.01.04 peach workload distribution methodology changed to dynamically redistribute workload and allow nested invocation. 
```

The limitations on nesting have been removed, so peach (and multi-threaded primitives) can be used inside peach.
To facilitate this, round-robin scheduling has been removed. Even though the initial work is still distributed in the same manner as before for compatibility, 

> the workload will be dynamically redistributed if a thread finishes its share before the others.

Reviewer: ```suggestion
the workload is dynamically redistributed if a thread finishes its share before the others.
```
